config/build.ts

```diff
@@ -40,31 +39,50 @@ const defaultApiSets = {
 };
 
 
-(() => {
-    Promise.resolve()
-        .then(processSnippets)
+(async() => {
```

Comment:
Right. I verified that it looks right in prod right after I pulled it to prod branch. It looks fine, I think.